### PR TITLE
add test coverage for core filters

### PR DESCRIPTION
**Pull request by relrod**
##### SUMMARY

Lots of tests. Gets coverage into the high 90's percent.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

test/integration/targets/filter_core

### CODE REVIEW

File: test/integration/targets/filter_core/tasks/main.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # Copyright: (c) 2019, Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
+# Note: |groupby is already tested by the `groupby_filter` target.
+
```
Comment (mattclay) [inline test/integration/targets/filter_core/tasks/main.yml:7]: Those tests should be moved into this one. Apparently I missed that during my test cleanup/reorg before migration.

I'd probably do that in a separate PR.

Comment (relrod) [reply test/integration/targets/filter_core/tasks/main.yml:7]: I agree, although I looked briefly and they do a separate `jinja2` install kind of like I had to do for `filter_urls`, so we'll have to port over that logic too.

**Review by mattclay — APPROVED** [review]
